tmail-backend/apps/postgres/src/main/java/com/linagora/tmail/james/app/PostgresTmailServer.java

```diff
@@ -252,8 +263,16 @@ private static Module chooseBlobStoreModules(PostgresTmailConfiguration configur
 
     public static Module chooseEventBusModules(PostgresTmailConfiguration configuration) {
         return switch (configuration.eventBusImpl()) {
-            case IN_MEMORY -> new DefaultEventModule();
+            case IN_MEMORY -> Modules.combine(new DefaultEventModule(),
+                new ActiveMQQueueModule(),
+                new MemoryEventStoreModule());
```

Comment:
We still can use `PostgresEventStoreModule` when memory event bus, no?

Reply:
I would tend to think if it's memory event bus, then event store should be memory too. But opened to discussion if you want

Reply:
Despite it being a memory event bus, we always have Postgres DB. Storing data structure in memory (instead of Postgres) could blow up TMail.

I think we should make it more useable than just for test in this case. e.g. Docaposte uses ActiveMQ so the memory event bus here does not always mean just for testing IMO.

Reply:
Alright alright. It makes sense and I'm willing to give it a shot. But in a follow up PR ok? Took long enough to get a green build with this :)

Reply:
=> https://github.com/linagora/tmail-backend/pull/997